{
	"id": 204700380,
	"body": "I updated go and build it from source again, running `GOTRACEBACK=all ./all.bash`, it failed but without an error again.\r\n\r\n```\r\n# ... (removed)\r\nok  \tmime/quotedprintable\t0.132s\r\nFAIL\r\nFAIL\tnet\t4.019s\r\nok  \tnet/http\t11.951s\r\n# ... (removed)\r\nok  \tcmd/vet\t2.100s\r\n2016/04/02 13:13:54 Failed: exit status 1\r\n```\r\n\r\nIs there a reason why `GOTRACEBACK` doens't get passed to the test command?\r\n\r\nThe full build log: [build3.txt](https://github.com/golang/go/files/200800/build3.txt).",
	"user": {
		"login": "Thomasdezeeuw",
		"id": 3159064,
		"type": "User",
		"site_admin": false
	},
	"created_at": "2016-04-02T11:55:16Z",
	"updated_at": "2016-04-02T11:55:16Z"
}
